packages/realm/src/app-services/User.ts

```diff
@@ -242,8 +269,9 @@ export class User<
    *
    * @returns An service client with methods to register and deregister the device on the user.
```

Comment:
Lets go ahead and add an `@deprecated` here with a link to https://www.mongodb.com/docs/atlas/app-services/reference/push-notifications/